### PR TITLE
docker alpha build

### PR DESCRIPTION
**Pull request by redoracle**
- dockerfile alpha build updated
- workflows on dispatch enabled and minor fix

### CODE REVIEW

**Review by TrevorBenson — COMMENTED** [review]
I'm confused.
@redoracle
Much of this appears to be changes I approved in the prior PR. Did they get rolled back or overwritten by git actions somehow after they were merged?

**Review by TrevorBenson — APPROVED** [review]
I'm going to approve and merge and try to track if the changes are properly applied or get overwritten at some stage.

File: .github/workflows/node-latest.yml
Comment (TrevorBenson) [inline .github/workflows/node-latest.yml:5]: This may conflict again. Maybe the alpha branch is not being pulled into the fork prior to submitting the next PR? This was resolved with an upstream merge. 

I find it much easier to rebase a branch inside guild-operators spawned from origin/alpha than it is to rebase my own fork of origin/alpha. Might make things simpler to change to using a branch as a collaborator for PR's vs. a repo fork as a community submission for PR's.

**Comment by redoracle** [comment]
> I'm confused.
> @redoracle
> Much of this appears to be changes I approved in the prior PR. Did they get rolled back or overwritten by git actions somehow after they were merged?

Sorry I am just closing and making a new PR every time :)

**Pull request closed without merging**
